releasenotes/releasenotes_V0.7.2.md

```diff
@@ -0,0 +1,93 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 improves the capabilities of the 0.7.0 release by adding ... 
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on a small set of parameters, like: `start`, `end`, and `timeframe`. 
+
+:sparkles: *Focused UI enhancements for Quality Gates*: The evaluation tile that displays the quality gate evaluation result has been improved in various ways. First, the Heatmap in this tile now highlights the currently selected evaluation result. Second, it also displays the evaluation results that were used for comparisons giving the user more insights into the quality gate. 
+
+:tada: *Usage statistics for Keptn installation*: If you want to get more insights into your Keptn installation, feel free to deploy the [statistics-service](https://github.com/keptn-sandbox/statistics-service) from the Keptn sandbox. This service provides usage statistics based on events and service executions.
+
+## Keptn Specification
+
+Implemented **Keptn spec** version: [0.1.6](https://github.com/keptn/spec/tree/0.1.6)
+
+## New Features
+
+<details><summary>Platform Support / Installer</summary>
+<p>
+
+- K8s 1.19 support [#2411](https://github.com/keptn/keptn/issues/2411)
+
+</p>
+</details>
+
+<details><summary>API</summary>
+<p>
+
+- Introduced API endpoint `\evaluation` for triggering evaluations [#2387](https://github.com/keptn/keptn/issues/2387)
+- Swagger automatically determines the scheme (https or http) [#2325](https://github.com/keptn/keptn/issues/2325)
+
+</p>
+</details>
+
+<details><summary>CLI</summary>
+<p>
+
+- The output of `keptn version` shows the Keptn API version [#2412](https://github.com/keptn/keptn/issues/2412)
+- Improve robustness and UX of `keptn generate support-archive` [#2340](https://github.com/keptn/keptn/issues/2340)
+- Point user to upgrade docs, fixed bug in `keptn update project` [#2293](https://github.com/keptn/keptn/issues/2293)
```

Comment:
let's also add https://github.com/keptn/keptn/issues/2445

Reply:
2445 is already contained.